lambda/aws-contentgen-appsync-lambda/src/image_generator.py

```diff
@@ -0,0 +1,283 @@
+import boto3,os,base64,json
```

Comment:
license header missing

Reply:
added in 3f90ec921befd1a503e473dd0a582f072c69f76b